src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,52 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'object_label': '',
+    'equivalence': '>A oboInOwl:source',
+    'ORCID': '>A oboInOwl:source SPLIT=|',
+}
+
+
+def sssom_to_robot_template(inpath: Union[str, Path], outpath: Union[str, Path]):
+    """Convert SSSOM to ROBOT template."""
+    msdf: MappingSetDataFrame = parse_sssom_table(inpath)
+    df: pd.DataFrame = msdf.df
+
+    # Filter
+    df = df[df['predicate_id'] == 'skos:exactMatch']
```

Comment:
FYI: filtering only exact matches

Reply:
I just realised that the mappings are very often not exact.. grrr. instead of filtering, can do ith like this:

if skos:exactMatch --> Mondo:equivalentTo in `equivalence` column
else if skos:relatedMatch --> Mondo:whateverTheRelatedToCode
else if skos:broadMatch --> Mondo:whateverTheMondoIsNarrowerThanObjectCodeIs
else --> do not include anything in the `equivalence` column, but keep the record.

Reply:
@matentzn Sounds good!

I was going to suggest that this code might fit better into `sssom-py`, but what with the `MONDO` related annotations, I guess not.

Reply:
@matentzn Are you sure about this one?
> else if `skos:broadMatch` --> `MONDO:mondoIsNarrowerThanSource`

---

I have my own notes about `skos:broadMatch` and `skos:narrowMatch`, where I've written:
> In a `skos:broadMatch` b, 'a' is broader than 'b'.

We're parsing `mondo-nando.sssom.tsv`, and the Mondo IDs are on the left (subject), and NanDO on the right (object). So for `skos:broadMatch` I should want `MONDO:mondoIsBroaderThanSource`, no?

I apologize, I know this is fundamental and I should know this.

As my source for this factoid, I have linked ChatGPT: https://chat.openai.com/share/3b6942a1-8152-4345-b1cd-763dd4403e2a

...which I realize could be totally wrong. The only reason I asked it is because the [skos docs](https://www.w3.org/TR/skos-reference/) are very hard to 

To make things more confusing, I've heard that `broadMatch` flips the order of `broader`, and `narrowMatch` flips the order of `narrower`, but I can't find any source that confirms this. What I do see is that these are sub-properties... and if that's the case, I'd imagine they probably would not be flipped.

It's easier to find documentation on `broader` and `narrower`, but the examples I find are conflicting.

(CC @twhetzel in case you happen to have a good solid understanding of this. Otherwise if Nico can give me a firm statement on this or link to a good source, you'd probably want to know too)

Reply:
Also you don't want `skos:narrowMatch` --> `MONDO:mondoIsBroaderThanSource`? For now, if that's the case, the cell will just be blank.

Reply:
Alright, this is done. FYI, `exactMatch` is the only one in our current set but I imagine that may change in the future.

```py
PRED_MAP = {
    'skos:relatedMatch': 'MONDO:relatedTo',
    'skos:exactMatch': 'MONDO:equivalentTo',
    'skos:broadMatch': 'MONDO:mondoIsNarrowerThanSource',
}
```

```
df['equivalence'] = df['predicate_id'].map(lambda pred: PRED_MAP.get(pred, ''))
```

Reply:
```
In a skos:broadMatch b, 'a' is broader than 'b'.
```

Check https://mapping-commons.github.io/sssom/mapping-predicates/#precision

"The object is conceptually broader than the subject.".

Your solution sounds correct to me!